### DISCUSSION TITLE
Ability to Duplicate and/or Export/Import Fieldtypes from the Field Manager

### DISCUSSION THREAD

**Discussion by mcfarlandonline** [ExpressionEngine/ExpressionEngine#2953] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I have to recreate common fieldtypes from scratch in the Field Manager. Often, I may have already created the fieldtype I want, I just need to change its Name + Short Name. 

When building a new website, I like to start out with 4 core fieldtypes. (3 are grid fields containing text or textarea fields, and 1 is just a textarea field). Then I create a fluid field to contain those 4 core fields. 

If the website is part of a Multiple Site Manager system or a multilingual website, I'll add the _**site's**_ Short Name to the _**field's**_ Short Name, but the fieldtypes remain the same. 

**Describe the solution you'd like**
I would love a way to duplicate existing fieldtypes in the Field Manager and/or be able to export them and then import them into the same or other websites. 

**Describe alternatives you've considered**
I can see a challenge being if the field contains file directory settings, or is a relationship field dependent on channel entries, but for some of the most basic fields (like textarea), this would still speed up things for me. 

Ideally, there'd be a prompt (when duplicating or importing a field) to change the field's Name + Short Name. If it's also possible to also duplicate/import more sophisticated fields, then the prompt could also include reconfiguring those field settings.

**Comment by robsonsobral** [comment]
A way to import and export only fields and their groups could be a good start!

**Comment by intoeetive** [comment]
Field Cloning has been added in ExpressionEngine 7.3.0
